### PR TITLE
Feat/ggircs app

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

**Review by dleard — CHANGES_REQUESTED** [review]
can we get rid of all the test/data/Report_xxx files?

File: ggircs-app/app/.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+SESSION_SECRET=blah
```
[the remark this answers is not in the document]
Comment (matthieu-foucault) [reply ggircs-app/app/.env:1]: This file was committed before adding the line in .gitignore, which is why it shows up

**Comment by matthieu-foucault** [comment]
The  `test/data/Report_xxx` files are used in cas-ciip-portal. We should probably have tests using them in this repository

File: ggircs-app/app/server/routers/sso.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  //   NO_AUTH ||
+  //   AS_ADMIN ||
+  //   AS_ANALYST ||
+  //   AS_PENDING ||
+  //   AS_REPORTER ||
+  //   AS_CERTIFIER ||
+  //   AS_CYPRESS
+  // ) {
+  //   return res.json(3600);
+  // }
```
Comment (pbastia) [inline ggircs-app/app/server/routers/sso.js:91]: we can probably delete this

**Review by pbastia — APPROVED** [review]

**Review by pbastia — APPROVED** [review]

**Pull request merged**